index.js

```diff
@@ -6,4 +6,4 @@ import {AppRegistry} from 'react-native';
 import App from './App';
 import {name as appName} from './app.json';
 
-AppRegistry.registerComponent(appName, () => App);
```

Comment:
I have renamed the app in the app.json file. So we don't need this here.

Reply:
without renaming I can't run the project